### PR TITLE
Vuex implementation 

### PR DESCRIPTION
**Pull request by lisandroiraguen**
with Mnemonic length form validation

### CODE REVIEW

**Comment by nothingalike** [comment]
@lisandroiraguen is this different than the other PR?

**Comment by lisandroiraguen** [comment]
yes, it has some changes

**Comment by nothingalike** [comment]
Do we need to merge both?

**Comment by lisandroiraguen** [comment]
it is the same, you can marge both.

File: src/store/wallet/actions.js [hunk 1]
```diff
@@ -1,8 +1,24 @@
...
+     var cardanoPath= require('path').resolve('./');
+     const cardanoPath2 =  path.resolve(cardanoPath, 'cardano') ;
+     const addressCli = path.resolve('.', cardanoPath2, process.platform, 'cardano-address');
+     var cardano_words = `"${addressCli}" recovery-phrase generate --size ${wordsNumber}`;    
+    
+     const { stdout, stderr } = await cmd(cardano_words);
+     if(stderr) throw stderr;      
+    
+     var words =stdout.replace('\n', '');
+     commit(types.SET_MNEMONIC, words);
```
Comment (nothingalike) [inline src/store/wallet/actions.js]: Alright so this isnt a viable decision. The only reason to move this 'type' operation to the Vue side is to support web/mobile capabilities, but we cannot use the binaries for such a move. We will need to remove the binary requirement and use this libary (most likely) https://github.com/vacuumlabs/cardano-crypto.js

**Review by nothingalike — CHANGES_REQUESTED** [review]

**Comment by nothingalike** [comment]
@lisandroiraguen idk if you want to keep this PR since we cannot move the binary logic to the vue side. ultimately we could move the call that is being made from the component here, right?

File: src/store/wallet/actions.js (hunk 1 above)
Comment (lisandroiraguen) [reply src/store/wallet/actions.js]: I see, don't marge it yet, I will use cardano-crypto.js there

Comment (lisandroiraguen) [reply src/store/wallet/actions.js]: I will change line 12 to 20.

**Comment by lisandroiraguen** [comment]
yes, that is correct, I am going to make some changes and I am going to push it again

**Comment by lisandroiraguen** [comment]
@kylejohns in this branch is not yet solved the mount event call problem that we have discussed, but I have moved the server request to vuex as a first step

**Pull request merged**